packages/shared/src/lib/query.ts

```diff
@@ -26,8 +26,8 @@ import {
 export enum OtherFeedPage {
   Tag = 'tag',
   Tags = 'tags',
-  Squad = 'squads',
-  SquadPage = 'squads[handle]',
+  Squad = 'squadsdiscover',
```

Comment:
I think we might do some eval somewhere, but not sure it's on this.
To check if page === some value it can render the name differently.

Just noting it so we keep a close eye on mobile reports after deploying :)

Reply:
I think the tests failed because of it, we have some string evals/types.

Reply:
@ilasw Yeah it also breaks on mobile now.
The squad is not shown as active at the moment.

Maybe best to revert this and re-look at it.